test/e2e/main_test.go

```diff
@@ -29,6 +30,8 @@ func TestMarketplace(t *testing.T) {
 	// Run Test Groups
 	if coAPIPresent {
 		t.Run("cluster-operator-status-test-group", testgroups.ClusterOperatorTestGroup)
+		// If CO API is available, assume proxy api is as well.
```

Comment:
Then do you really need this variable. You might as well treat `coAPIPresent` as `onOpenShiftCluster`

Reply:
This will still be needed, please read above.

Reply:
Perhaps naming the function `EnableProxyTests` would better portray the point of this function.

Reply:
I am asking if you need the `proxyAPIAvailable` variable.

Reply:
We do, because the `coAPIPresent` in the `e2e` package cannot be referenced from the `testgroups` package.